app/globals.css

```
@@ -262,11 +262,23 @@ h1, h2, h3 {
   animation: typingDot 1.1s ease-in-out infinite;
 }
 
+/* Marching-ants dashed border used by the import entity selector. */
+@keyframes marchingAnts {
```

Stylelint (17.14.0)

[error] Expected keyframe name "marchingAnts" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)